src/Microsoft.VisualStudio.AppDesigner/PropPageDesigner/PropPageDesignerEditorFactory.vb

```diff
@@ -121,6 +122,11 @@ Namespace Microsoft.VisualStudio.Editors.PropPageDesigner
                     DesignerLoader.Dispose()
                 End If
 
+                ' If we created the doc data then we should dispose it for a failure
+                If ExistingDocData Is Nothing Then
```

Comment:
We always create the DocData, so I'm not sure I understand this comment, or why this line is needed.

Reply:
I don't think we should ever explicitly dispose, and instead make `IVsPersistDocData.Close` do it and it appears to be explicitly called by the RDT when its done with.

Reply:
> We always create the DocData, so I'm not sure I understand this comment, or why this line is needed.

We only create the doc data here if there isn't an ExistingDocData passed in. This line is just cleaning up if things fail to initialize under the assumption that in such a failed case, the doc data wouldn't end up in the RDT so could never be cleaned up via its removal.